charts/pega/README.md

```diff
@@ -672,6 +672,15 @@ Parameter     | Description | Default value
 `trustStorePassword` | If required provide trustStorePassword value in plain text. For better security leave this parameter blank and include the password in an external secrets manager with the key CASSANDRA_TRUSTSTORE_PASSWORD. | Empty
 `keyStore` | If required, provide the keystore certificate file name.<br/>When using a keystore certificate, you must also include a Kubernetes secret name that contains the keystore certificate in the global.certificatesSecrets parameter.<br/>Pega deployments only support keystores that use the Java Key Store (.jks) format. | Empty
 `keyStorePassword` | If required provide keyStorePassword value in plain text. For better security leave this parameter blank and include the password in an external secrets manager with the key CASSANDRA_KEYSTORE_PASSWORD. | Empty
+`asyncProcessingEnabled*` | Specify a prefix to use when creating Pega-managed keyspaces in Cassandra. | false
```

Comment:
is this correct? I see "Specify a prefix to use when creating Pega-managed keyspaces in Cassandra." is also used to describe keyspacesPrefix. maybe copy error?

Reply:
Done